db/migrate/20231207104401_move_programme_observation.rb

```diff
@@ -0,0 +1,18 @@
+class MoveProgrammeObservation < ActiveRecord::Migration[6.0]
+  def up
+    Observation.where(observation_type: :programme).find_each do |observation|
+      observation.obserable = observation.programme
```

Comment:
There's a typo here I think?

Reply:
Oh gosh, good spot, thank you. Strange that it ran (or perhaps I changed it afterwards).